docs/getting_started.md

```diff
@@ -2,17 +2,22 @@
 
 This section provides information on next steps you might take after downloading a dataset from the ScPCA portal.
 
-We have also provided a workflow that can be used to perform initial filtering, dimensionality reduction, normalization, and clustering on single-cell and single-nuclei samples in parallel.
-This workflow and more resources for processing single-cell and single-nuclei datasets can be found on our Github in the [`scpca-downstream-analyses` repository](https://github.com/AlexsLemonade/scpca-downstream-analyses).
+We also have a separate GitHub repository that contains workflows for some common analysis performed on single-cell RNA-sequencing data.
+These workflows are designed to apply the same analysis (e.g., clustering) across multiple samples in parallel.
+These workflows and more resources for processing single-cell and single-nuclei datasets can be found in the [`scpca-downstream-analyses` repository](https://github.com/AlexsLemonade/scpca-downstream-analyses).
 
 ## Importing ScPCA data into R
 
 Quantified single-cell or single-nuclei gene expression data is provided as an RDS file as described in the {ref}`single cell gene expression file contents section<sce_file_contents:single-cell gene expression file contents>`.
-There are two RDS files that will be available for each library, a `filtered.rds` and an `unfiltered.rds` file.
-The `filtered.rds` files will contain the gene expression data for only droplets that are likely to contain cells, removing any potential empty droplets.
-The `unfiltered.rds` file will contain the gene expression data for all droplets, regardless of the presence of a cell or not.
+There are three RDS files that are available for each library: an `_unfiltered.rds`, a `filtered.rds`, and a `processed.rds` file.
```

Comment:
I removed the lone underscore (which I may have inserted). I couldn't decide if we shoudl ahve those, but we don't use them in most places, so better to just leave them all off.
```suggestion
There are three RDS files that are available for each library: an `unfiltered.rds`, a `filtered.rds`, and a `processed.rds` file.
```